frontend/src/metabase/components/Popover/TippyPopover.info.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+
+import TippyPopover from "./TippyPopover";
+
+export const component = TippyPopover;
+export const description = "Wrapper around react-popper";
+
+const style = {
+  border: "1px solid black",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+};
+
+const content = (
```

Comment:
Any reason not to use styled components in this file?

Reply:
Pure laziness 🙂 . I cleaned it up a bit in https://github.com/metabase/metabase/pull/19042/commits/e9979fd8c779ce2b6bae43713076d563fe96a130